components/brave_wallet/browser/wallet_data_files_installer.cc

```diff
@@ -172,122 +172,108 @@ void OnSanitizedCoingeckoIdsMap(data_decoder::JsonSanitizer::Result result) {
       std::move(*coingecko_ids_map));
 }
 
-void HandleParseCoingeckoIdsMap(base::FilePath absolute_install_dir,
-                                const std::string& filename) {
-  const base::FilePath coingecko_ids_map_json_path =
-      absolute_install_dir.AppendASCII(filename);
-  std::string coingecko_ids_map_json;
-  if (!base::ReadFileToString(coingecko_ids_map_json_path,
-                              &coingecko_ids_map_json)) {
-    VLOG(1) << "Can't read coingecko ids map file: " << filename;
+void OnSanitizedOfacAddressesList(data_decoder::JsonSanitizer::Result result) {
+  if (!result.has_value()) {
+    VLOG(1) << "OFAC addresses list JSON validation error:" << result.error();
     return;
   }
 
-  data_decoder::JsonSanitizer::Sanitize(
-      std::move(coingecko_ids_map_json),
-      base::BindOnce(&OnSanitizedCoingeckoIdsMap));
+  absl::optional<std::vector<std::string>> list =
+      ParseOfacAddressesList(*result);
+  if (!list) {
+    VLOG(1) << "Can't parse ofac addresses list.";
+    return;
+  }
```

Comment:
We may want this to fail open rather than fail closed here. I suspect there may be cases where people block the list via the request from component updater/delete it from filesystem in which case I think that would break the wallet.

Reply:
If the file is gone or can't be parsed, we just log it and move on.  The list of OFAC addresses remains an empty set, so the wallet should still be usable.

Reply:
Thanks for clarifying that, I was thinking that return would be a failure but misread that logic.